slave.mk

```diff
@@ -1447,6 +1447,11 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 			$(if $(shell ls files/build_templates/$($(docker:-dbg.gz=.gz)_CONTAINER_NAME).service.j2 2>/dev/null),\
 				$(eval $(docker:-dbg.gz=.gz)_GLOBAL = yes)
 			)
+
+			if [ x$($(docker:-dbg.gz=.gz)_CONTAINER_NAME) == x"gnmi" ]; then
+				j2 files/build_templates/telemetry.service.j2 > telemetry.service
```

Comment:
>telemetry [](http://example.com/codeflow?start=52&length=9)

Should we rename service name in order to align with container name?

Reply:
Now we only have one docker image for gnmi, and we have two service: gnmi and telemetry, telemetry is disabled by default.

Reply:
The current reality is one docker image, two docker containers, two services. The build infra has some difficulty to handle this mapping.

On proposal: use two docker image, two docker containers, two services. You may construct the telemetry image FROM gnmi image, with nothing extra added. Then the current build infra will easily support it.

Reply:
Great suggestion!